.bazelrc

```diff
@@ -3,7 +3,7 @@ try-import ci.bazelrc
 try-import .bazeldnf/sandbox.bazelrc
 
 # Set common values for all builds (run, test and coverage inherit from build)
-build --stamp --workspace_status_command=./hack/print-workspace-status.sh --host_force_python=PY3
+build --stamp --noenable_bzlmod --workspace_status_command=./hack/print-workspace-status.sh --host_force_python=PY3 --host_linkopt=-lstdc++
```

Comment:
do we have an issue working on enabling the bzlmod?

Reply:
Actually I think I can remove this from here - its only relevant when we move to bazel v7. I will remove it.